src/pages/Join/CreateAccount.js

```diff
@@ -0,0 +1,259 @@
+import React, { useState } from 'react';
+import Container from '../../components/AccountContainer/components/Container';
+import SocialBtnGroup from '../../components/AccountContainer/components/SocialBtnGroup';
+import Title from '../../components/AccountContainer/components/Title';
+import Input from '../../components/AccountContainer/components/Input';
+import styled from 'styled-components/macro';
+import Button from '../../components/Button/Button';
+import Divider from '../../components/AccountContainer/components/Divider';
+import { Link, useHistory } from 'react-router-dom';
+import Section from '../../components/AccountContainer/components/Section';
+import { API } from '../../config';
+
+function CreateAccount() {
+  const history = useHistory();
+  const [inputs, setInputs] = useState({
+    username: '',
+    email: '',
+    password: '',
+  });
+  const [secondaryInput, setSecondaryInput] = useState({
+    secondaryEmail: '',
+    secondaryPassword: '',
+  });
+  const [isChecks, setIsChecks] = useState({
+    all: false,
+    check1: false,
+    check2: false,
+    check3: false,
+    check4: false,
+  });
+
+  const handleTextInput = e => {
+    const { name, value } = e.target;
+    setInputs({
+      ...inputs,
+      [name]: value,
+    });
+  };
+
+  const handleSecondaryInput = e => {
+    const { name, value } = e.target;
+    setSecondaryInput({
+      ...secondaryInput,
+      [name]: value,
+    });
+  };
+
+  const handleCheckInput = e => {
+    const { name, checked } = e.target;
+    setIsChecks({
+      ...isChecks,
+      [name]: checked,
+    });
+  };
+
+  const allCheck = () => {
+    const { check1, check2, check3 } = isChecks;
+    const isChecked = check1 && check2 && check3;
+    if (!isChecked) {
+      setIsChecks({
+        check1: true,
+        check2: true,
+        check3: true,
+        check4: true,
+      });
+    } else if (!check1 || !check2 || !check3) {
+      setIsChecks({
+        ...isChecks,
+        all: false,
+      });
+    } else {
+      setIsChecks({
+        check1: false,
+        check2: false,
+        check3: false,
+        check4: false,
+      });
+    }
+  };
+
+  const handleSignUp = () => {
+    if (username && email && password && isEqual && isChecked) {
```

Comment:
```suggestion
  const handleSignUp = () => {
    if (!(username && email && password && isEqual && isChecked)) return;
```
early return을 사용하면 이하 내용들은 if문으로 인해 한 depth 들어가지 않아도 됩니다

Reply:
<img width="703" alt="스크린샷 2021-06-01 오후 4 53 59" src="https://user-images.githubusercontent.com/77728308/120287167-f780f100-c2f9-11eb-95b9-9b5302ba0382.png">
이렇게 하면 될까요??